x-pack/plugins/triggers_actions_ui/public/application/sections/actions_connectors_list/components/actions_connectors_list.tsx

```diff
@@ -346,8 +346,8 @@ export const ActionsConnectorsList: React.FunctionComponent = () => {
 
   return (
     <section data-test-subj="actionsList">
-      <DeleteConnectorsModal
-        callback={(deleted?: string[]) => {
+      <DeleteModalConfirmation
+        callback={async (deleted?: string[]) => {
```

Comment:
not a blocker, but could we change `callback` to something that tells us when and how it gets called?
The name `callback` is a bit generic and makes this harder to use and maintain, as i have to go look at the details to understand what it is.

Perhaps we can have two callbacks, `onCanceled` and `onDeleted`? That way there's an explicit callback for each outcome, so it's easier for users of the component to handle the two cases differently in the future.
At the moment you need to *know* that when cancel is called you get no arguments, but that seems both easy to get wrong and could easily change and unintentionally break someone else's usage of the component.